### PR TITLE
SKantor SOL

### PR DESCRIPTION
**Pull request by KantorSerhiy**

### CODE REVIEW

File: db/models.py [hunk 1]
```diff
@@ -45,3 +54,64 @@ class MovieSession(models.Model):
...
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(to=MovieSession,
+                                      on_delete=models.CASCADE)
+    order = models.ForeignKey(to=Order,
+                              on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        # unique_together = ["row", "seat", "movie_session"]
```
[the remark this answers is not in the document]
Comment (KantorSerhiy) [reply db/models.py]: No, i just tried to see if it will work

**Comment by KantorSerhiy** [comment]
do i have to fix all the mistakes flake8? the first time they were not there.

File: db/models.py [hunk 2]
```diff
@@ -45,3 +54,63 @@ class MovieSession(models.Model):
...
+            raise ValidationError(
+                {'row': f"row number must be in available range:"
+                        f" (1, rows): ("
+                        f"1, {self.movie_session.cinema_hall.rows})"})
+        if not (1 <= self.seat <= self.movie_session.cinema_hall.seats_in_row):
+            raise ValidationError(
+                {'seat': f"seat number must be in available range:"
+                         f" (1, seats_in_row): (1, "
+                         f"{self.movie_session.cinema_hall.seats_in_row})"})
+
```
Comment (Abnormaltype) [inline db/models.py:102]: use here "" instead of ''

File: settings.py [hunk 3]
```diff
@@ -25,4 +25,31 @@
...
+        }
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.'
+                'CommonPasswordValidator',
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.'
+                'NumericPasswordValidator',
+    },
```
Comment (Abnormaltype) [inline settings.py:54]: and here

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
also add type annotation , only for your code

**Review by Abnormaltype — APPROVED** [review]
Well done!

**Pull request closed without merging**